doc/code/qml_ops.rst

```diff
@@ -0,0 +1,13 @@
+qml.ops
+=======
+
+Functions
+---------
```

Comment:
This name is actually a bit confusing in this context because module members are often included under a `Functions` and a `Classes` header, but in this case the header refers to the module name.